core/src/main/java/org/p2p/core/pricecache/PriceCacheRepository.kt

```diff
@@ -0,0 +1,25 @@
+package org.p2p.core.pricecache
+
+import java.math.BigDecimal
+import org.p2p.core.model.TokenPriceWithMark
+
+class PriceCacheRepository() {
+
+    private val cachedPrices: MutableMap<String, TokenPriceWithMark> = mutableMapOf()
```

Comment:
Just pass <TokenId, TokenPriceWithMark> 
or rename variable to tokenIdsToPrices, `String` is just to ambiguous

Reply:
because it is not ID!

Reply:
it is key (ID or Address what you will choose as a key for map)

Reply:
strange, i thought it's id
what is it then? mint?

Reply:
understood, thanks!

Reply:
contractId for Ethereum prices

 override suspend fun getTokenPrices(
        tokenAddresses: List<String>,
        targetCurrency: String
    ):